docs/firefox-stub-attribution.rst

```diff
@@ -24,15 +24,43 @@ page of the website. The script only runs on Windows (since the stub installer i
 Windows feature) and if cookies are enabled. The script also respects Do Not Track,
 so it will not run if the user agent has this flag set.
 
-The stub attribution script looks for ``utm_*`` params on the page URL as well as the
-referrer, and then creates a hash of that data by passing it to an authentication
-service in bedrock. This hash is then accompanied by a signed, encrypted signature
-to prove that the data came from a trusted source. Both pieces of authenticated
-data are then stored in a cookie.
+The stub attribution script looks for ``utm_*`` params on the page URL, as well
+``document.referrer`` and various other values, and then creates a hash of that
+data by passing it to an authentication service in bedrock. This hash is then
+accompanied by a signed, encrypted signature to prove that the data came from a
+trusted source. Both pieces of authenticated data are then stored in a cookie.
```

Comment:
**question (non-blocking)**: does this cookie expire? is it worth mentioning how long it is stored?

Reply:
good suggestion, will add 👍